src/environment/environment.repository.ts

```diff
@@ -20,7 +20,7 @@ export class EnvironmentRepository {
   }
 
   async setNewValues(dataType: DataType, newVals: NewValsDTO) {
-    await this.prismaService.dataPointThresholds.upsert({
+    return await this.prismaService.dataPointThresholds.upsert({
       where: { dataType: dataType },
```

Comment:
Redundant to explicitly specify key and value

```suggestion
      where: { dataType },
```